src/evidently/collector/app.py

```diff
@@ -89,25 +87,25 @@ async def reference(
     return {}
 
 
-@post("/{id:str}/data")
-async def data(
+@post("/{id:str}/data", sync_to_thread=True)
+def data(
     id: Annotated[str, Parameter(description="Collector ID")],
-    request: Request,
-    service: CollectorServiceConfig,
-    storage: CollectorStorage,
+    data: Any,
```

Comment:
This achieves the same as calling `await request.json()`, but allows using a synchronous function